src/main/scala/com/fulcrumgenomics/umi/DuplexConsensusCaller.scala

```diff
@@ -161,15 +162,17 @@ class DuplexConsensusCaller(override val readNamePrefix: String,
         // Check for this explicitly here.
         if (singleStrand1.nonEmpty) {
           val ss1Flag = singleStrand1.head.negativeStrand
-          val ss1MI   = singleStrand1.head.apply[String](ConsensusTags.MolecularId)
-          require(singleStrand1.forall(_.negativeStrand == ss1Flag),
-            s"Not all AB-R1s and BA-R2s were on the same strand for molecule with id: $ss1MI")
+          if (singleStrand1.exists(_.negativeStrand != ss1Flag)) {
+            rejectRecords(ab ++ ba, FilterCollision)
+            return Nil
+          }
         }
         if (singleStrand2.nonEmpty) {
           val ss2Flag = singleStrand2.head.negativeStrand
-          val ss2MI   = singleStrand2.head.apply[String](ConsensusTags.MolecularId)
-          require(singleStrand2.forall(_.negativeStrand == ss2Flag),
-            s"Not all AB-R2s and BA-R1s were on the same strand for molecule with id: $ss2MI")
+          if (singleStrand2.exists(_.negativeStrand != ss2Flag)) {
+            rejectRecords(ab ++ ba, FilterCollision)
```

Comment:
Doing this here and above causes a line to be logged at the end with the count, right?